### PR TITLE
fix 970

### PR DESCRIPTION
**Pull request by jj105**
# 970

### CODE REVIEW

**Comment by devbtech** [comment]
works as expected

File: girderformindlogger/api/v1/applet.py [hunk 1]
```diff
@@ -533,7 +533,7 @@ def setConstraints(self, folder, activity, schedule, **kwargs):
             destName='applet'
         )
         .param(
-            'get_all_events',
+            'getAllEvents',
```
[the remark this answers is not in the document]
Comment (jj105) [reply girderformindlogger/api/v1/applet.py:536]: all request parameters are written using camelCase.

[the remark this answers is not in the document]
Comment (devbtech) [reply girderformindlogger/api/v1/applet.py:536]: in this case, is this an internal code style of the project?

[the remark this answers is not in the document]
Comment (jj105) [reply girderformindlogger/api/v1/applet.py:536]: I think so.

**Pull request merged**